docs/developer-guide/query-with-arrays-and-maps.md

```diff
@@ -12,9 +12,8 @@ declare and access these types.
 The following example shows how to create a ksqlDB table from an
 {{ site.aktm }} topic that has array and map fields. Also, it shows how to
 run queries to access the array and map data. It assumes a Kafka topic,
-named `users`. To see this example in action, create the `users` topic
-by following the procedure in
-[Write Streaming Queries Against {{ site.aktm }} Using ksqlDB](../tutorials/basics-docker.md).
+named `users` and uses the example data from the ksql-datagen tool. For more
```

Comment:
Would it be easier to redirect this and the structured data page to `https://docs.ksqldb.io/en/latest/how-to-guides/query-structured-data/`?